contrib/cirrus/setup_environment.sh

```diff
@@ -123,15 +128,16 @@ case "$PRIV_NAME" in
         if [[ "$TEST_ENVIRON" == "container" ]] && ((container)); then
             # There's no practical way to detect userns w/in a container
             # affected/related tests are sensitive to this variable.
-            _suns='export SKIP_USERNS=1'
-            echo "$_suns" >> /etc/environment
-            source /etc/environment
+            warn "Disabling usernamespace integration testing"
+            _suns='SKIP_USERNS=1'
+            echo "$_suns" >> /etc/ci_environment
+            source /etc/ci_environment
```

Comment:
All these `echo / source` -- why? Why not just `SKIP_USERNS=1` or `ENV=VALUE`? Unless I'm missing something big (and I've looked very hard), the entire purpose of lines 198-202 at the end is to write all that environment. It really bothers me to see all this convolution, because it makes me wonder, what am I missing? 

I would really like to see all of these simplified to just `ENV=whatever`. If there is an important reason why it is necessary to define an intermediate variable which is used in only one place, then (riskily) append to a global /etc file, then source that file, I would suggest adding a big fat comment to each instance -- or, better yet, rethinking it and writing it in a clean readable way.

Reply:
Cleaned up.